nx-dev/feature-doc-viewer/src/lib/doc-viewer.tsx

```diff
@@ -140,10 +140,9 @@ export function DocViewer({
                   aria-hidden="true"
                   href={[
                     'https://github.com/nrwl/nx/blob/master',
-                    document.filePath.replace(
-                      'nx-dev/nx-dev/public/documentation',
-                      'docs'
-                    ),
+                    document.filePath
+                      .replace('nx-dev/nx-dev/public/documentation', 'docs')
```

Comment:
I'm not sure if both are needed, tbh, but I just wanted to make sure I don't remove existing behaviour, but also catch this new thing.